package.json

```diff
@@ -108,7 +108,7 @@
     "axe-core": "4.2.3",
     "cypress": "7.6.0",
     "cypress-axe": "0.12.2",
-    "cypress-cucumber-preprocessor": "4.1.2",
+    "cypress-cucumber-preprocessor": "^4.0.0",
```

Comment:
Don't think you meant to do this?

Reply:
I think it came when I was doing all these weird manipulation with npm :/